csharp/src/Ice/LocatorInfo.cs

```diff
@@ -332,7 +339,8 @@ async Task<IReadOnlyList<Endpoint>> PerformGetAdapterEndpointsAsync(Reference re
                     }
                 }
             }
-            return task!;
+
+            return await task!.WaitAsync(cancel).ConfigureAwait(false);
```

Comment:
`!` seems is not required, what about `FindObjectByIdAsync` shouldn't we pass the cancel toke to it too?

Reply:
I'll remove the unnecessary !. No cancel token for FindObjectByIdAsync for the same reasons as above.